portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div id="gallery">
+      <h1>Here are some pictures that I like</h1>
+
+      <div class="image-thumbnails">
+        <a href="images/image-1.JPG"><img src="images/image-1.JPG"/></a>
+        
+        <a href="images/image-2.JPG"><img src="images/image-2.JPG"/></a>
+        <a href="images/image-3.jpg"><img src="images/image-3.jpg"/></a>
```

Comment:
nit: the extensions here are inconsistent. Some are ".JPG" while others are ".jpg".